src/huntsman/pocs/dome/musca.py

```diff
@@ -234,13 +200,12 @@ def __del__(self):
 
     def _write_musca(self, cmd, log_message=None):
         """Wait for the command lock then write command to serial bluetooth device musca."""
-        with self._command_lock:
```

Comment:
why remove the lock here?

Reply:
Yeah, the built in one second delay after write versus a lock seems odd.

Reply:
I've moved the lock so we are blocking for the whole write/read for the `status` property. It is also implemented on write for `open` and `close` methods.